cli/command/formatter/stats.go

```diff
@@ -25,7 +25,9 @@ const (
 
 // StatsEntry represents represents the statistics data collected from a container
 type StatsEntry struct {
+	Container        string
 	Name             string
+	ID               string
```

Comment:
yep, that's it. I kept `Container` as the user input as this is what it's doing now.